modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationChanger.java

```diff
@@ -150,6 +154,8 @@ private static class StorageRoots {
         /** Full storage data. */
         private final Data data;
 
+        private final Collection<String> ignoredKeys;
```

Comment:
I don't understand why it needs to be here, let's discuss it when you finish adding a new test